src/ucp/core/ucp_request.c

```diff
@@ -328,7 +328,7 @@ ucp_request_send_start(ucp_request_t *req, ssize_t max_short,
         } else {
             req->send.uct.func        = proto->bcopy_multi;
             req->send.tag.message_id  = req->send.ep->worker->tm.am.message_id++;
-            req->send.tag.am_bw_index = 0;
+            req->send.tag.am_bw_index = 1;
```

Comment:
are we sure here we have at least 2 am_bw lanes? do we need an assertion>

Reply:
@yosefe in case if there is only one lane - it will always return 0 lane index (the same initialization is used in https://github.com/openucx/ucx/blob/8b78dd5312023c15355c43a84bd7f4c1bb3f8d88/src/ucp/core/ucp_request.inl#L539)